Controller/DbControllers/SongController.cs

```diff
@@ -24,8 +24,9 @@ public List<Song> SearchSongsOnString(List<string> searchterms)
 
             var songs = GetList(); /*Context.Songs.AsEnumerable();*/
             List<Song> searchSongs = songs
-                .Where(song => searchterms.Any(s => song.Name != null && song.Name.ToLower().Contains(s.ToLower())) ||
-                               searchterms.Any(s => song.Artist != null && song.Artist.ToLower().Contains(s.ToLower())))
+                .Where(song => (searchterms.Any(s => song.Name != null && song.Name.ToLower().Contains(s.ToLower())) ||
```

Comment:
Slaan we statussen als een string op?
Waarom niet als een enum, zodat deze geen typfouten kan bevatten?

Reply:
goeie, heb het omgezet naar een enum

Reply:
Nice :D